itwin-platform-access/imodels-access-backend/package.json

```diff
@@ -60,5 +60,14 @@
     "@itwin/core-backend": "^4.0.0",
     "@itwin/core-bentley": "^4.0.0",
     "@itwin/core-common": "^4.0.0"
+  },
+  "pnpm": {
```

Comment:
do we need all these overrides? or can they be resolved with updating lockfile?

Reply:
also, adding overrides to a rush repo via pkg doesnt actually do anything

Reply:
I added these overrides to resolve high severity issues found by the audit command, it was blocking the build. 
Rush is configured here with pnpm, so these overrides are respected. I can take a look if it's possible to update the primary dependencies without breaking anything.

Reply:
I would recommend first reverting these changes, and running `rush update --full` to update the lockfile, and then run the audit again to see if this is still needed.

If they are, instead of applying the same overrides in every package.json, they should be added in a [pnpm-config.json](https://rushjs.io/pages/configs/pnpm-config_json/)